src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -124,26 +127,28 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $builder->setTimeout(null);
         $process = $builder->getProcess();
 
-        if (OutputInterface::VERBOSITY_VERBOSE > $output->getVerbosity()) {
+        if (OutputInterface::VERBOSITY_VERBOSE > $stdout->getVerbosity()) {
             $process->disableOutput();
         }
 
         $this
             ->getHelper('process')
-            ->run($output, $process, null, null, OutputInterface::VERBOSITY_VERBOSE);
+            ->run($stdout, $process, null, null, OutputInterface::VERBOSITY_VERBOSE);
```

Comment:
Hm since the OutputStyle has no idea of ConsoleOutput, which knows the stderr, the process helper actually really needs the original ConsoleOutput. Otherwise the process data would not be written to stderr but to stdout.